test/integration/XhtmlTest.php

```diff
@@ -18,6 +18,9 @@
 *               2009-2012 (update and modification) Public Research Centre Henri Tudor (under the project TAO-SUSTAIN & TAO-DEV);
 *
 */
+
+namespace oat\taoItems\test\integration;
```

Comment:
This test is totally broken because DOM* classes are not imported. Also it is old TaoUnitTestRunner, finally the className does not match the file name. Couldu fix it please

Reply:
It should work even if we don't import them, shouldn't we ? AFAIK the only advantage is a small improvement in term of performance or am I missing something ?

Reply:
You're right. Fixing it.